README.md

```diff
@@ -54,46 +54,66 @@ We recommend that you link to `<url for repository's GitHub pages>/workshop/HOME
 ### Manually triggering issue creation with GitHub Actions
 
 If you would like to automatically create issues corresponding to the tasks that are required for customizing this repository, you can manually trigger a workflow once you have created your new repo.
-Navigate to `Actions` and select `Manually trigger issue creation for standard set up` from under `All workflows`. 
+Navigate to `Actions` and select `Manually trigger issue creation for standard set up` from under `All workflows`.
 Use the `Run workflow` drop down menu; you will be required to input the appropriate `training-modules` release tag for your training workshop.
 
-## Local development 
+## Local development
 
-It can be helpful to build the GitHub pages site locally to check that passing parameter values is working as expected. 
-The following instructions are for Mac OS and have only been used on Mojave. 
+It can be helpful to build the GitHub pages site locally to check that passing parameter values is working as expected.
 
-1. Make sure you have [Bundler](https://bundler.io/) installed. Installation can be accomplished with `gem install bundler`.
- 
-  If you encounter a permissions error, it was solved (using this StackOverflow post as a guide) with the following on Mojave with [Homebrew](https://brew.sh/):
+### Installing GitHub pages dependencies locally
 
-  ```
-  # install rbenv
-  brew update
-  brew install rbenv
+Installing the dependencies for GitHub pages is best done in a separate ruby environment, managed by [rbenv](https://github.com/rbenv/) and [Bundler](https://bundler.io)
 
-  # add to ~/.bash_profile
-  echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
-  ```
+The following instructions are for installation on Mac OS.
+1. [Install rbenv](https://github.com/rbenv/rbenv#installation).
+The easiest, and recommended, installation is through Homebrew:
```

Comment:
👀 🍺

Reply:
Do we want to include a "but first install homebrew" as needed? For Data Lab folks this seems implicit, but in case other teams use our materials to set up a workshop it seems more helpful for them.
Link - https://brew.sh/

Reply:
```suggestion
The easiest, and recommended, installation is through [Homebrew](https://brew.sh/):
```